LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix2-expected.html

```diff
@@ -38,22 +38,22 @@
 <body>
 
 <section>
-<grid>
+<grid style="grid-template-columns: 15px auto">
   <item style="height:2ch">1</item>
-  <item style="grid-column: span 2">2 2</item>
-  <item style="grid-column: span 2">3 3</item>
+  <item>2 2</item>
+  <item>3 3</item>
   <item>4</item>
-  <item style="grid-column: span 2">5 5</item>
+  <item>5 5</item>
 
   <item class="hidden" style="grid-area: 4/2">0 0</item>
 </grid>
 
-<grid>
```

Comment:
Why does the first column here have to be a fixed value? Why is it simply not based off the size of the content?